Alexander_Putman.cfdg

```diff
@@ -0,0 +1,68 @@
+startshape CENTER
+
+rule CENTER {
+	RANDOMCIRCLE {}
+	MOVE {}
+}
+
+rule MOVE {
+	RANDOMCIRCLE {}
+	MOVE {x 4 y 2}
+}
```

Comment:
Interesting! What was the motivation (if any) behind the specific moves in these rules?